src/modules/roc_audio/resampler_updater.cpp

```diff
@@ -96,13 +98,16 @@ bool ResamplerUpdater::update(packet::timestamp_t time) {
         update_time_ += update_interval_;
     }
 
+    const float adjusted_coef = float(sample_rate_coef_ * fe_.freq_coeff());
```

Comment:
Why do we need a cast?

Reply:
o my gosh, at the beginning I had a crazy idea, that we can overflow here, but after your comment, I've reread the code. Both values are near about 1, we should throw this cast.

Reply:
I don't get it. If both arguments are floats, this cast simply does nothing.